src/components/app/NavBar/index.tsx

```diff
@@ -1,109 +1,89 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import { Link as RouterLink } from 'react-router-dom';
-import { makeStyles } from '@material-ui/core/styles';
-import { useLocation } from 'react-router-dom';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import IconButton from '@material-ui/core/IconButton';
 import Container from '@material-ui/core/Container';
+import { Grid, Typography } from '@material-ui/core';
 import styled from 'styled-components';
 import Brightness2Icon from '@material-ui/icons/Brightness2';
 import Brightness4Icon from '@material-ui/icons/Brightness4';
-import logoYearnLight from '../../../images/logo_yearn_watch_light.svg';
-import logoYearnDark from '../../../images/logo_yearn_watch_dark.svg';
+import logoRoboVaultLight from '../../../images/logo_robo_vault.png';
+import logoRoboVaultDark from '../../../images/logo_robo_vault.png';
 import Link from '@material-ui/core/Link';
-import NetworkSelect from '../../common/NetworkSelect';
-import { Network } from '../../../types';
 
 const StyledAppBar = styled(AppBar)`
     background-color: transparent !important;
 `;
 
-const StyleNetworkSelect = styled.div`
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    height: 1.8rem;
-    width: 9rem;
-`;
-
 const StyledImg = styled.img`
     height: 68px;
     @media (max-width: 768px) {
         height: 40px;
     }
 `;
+
+const StyledWatchTitle = styled(Typography)`
+    color: ${({ theme }) => theme.iconTheme};
+    font-family: 'Lucida Console', 'Courier New', monospace;
+    font-weight: bolder;
+`;
+
 const StyledBrightness2Icon = styled(Brightness2Icon)`
     color: ${({ theme }) => theme.iconTheme};
 `;
 
 interface NavBarProps {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     themeToggler: any;
-
     theme: string | boolean | (() => void);
 }
-const useStyles = makeStyles((theme) => ({
-    root: {
-        flexGrow: 1,
-    },
 
-    img: {
-        flexGrow: 1,
-        textAlign: 'center',
-
-        [theme.breakpoints.up('md')]: {
-            marginLeft: '15%',
-        },
-    },
-}));
 export const NavBar: React.FC<NavBarProps> = ({ themeToggler, theme }) => {
```

Comment:
This file is modified so I could add the RoboVault logo to the Navbar and to remove the network dropdown. I cleaned up the use of the flexbox toolbar to leverage native MaterialUI components over custom styled components.